plugins/scaffolder-backend/src/scaffolder/tasks/TaskWorker.ts

```diff
@@ -48,6 +48,11 @@ export class TaskWorker {
       return JSON.stringify(parseRepoUrl(repoUrl));
     });
 
+    this.handlebars.registerHelper('projectSlug', repoUrl => {
```

Comment:
I put that here because I didn't find a "nice way" to do that in the template file, I accept suggestions about that

Reply:
I it not always `owner` for GitHub? or in general deterministic for each SCM provider?

Reply:
I think the only difference is Azure DevOps, but I really don't get how that works.
https://github.com/backstage/backstage/blob/master/plugins/scaffolder/src/components/fields/RepoUrlPicker/RepoUrlPicker.tsx#L40

Should I remove the organization param and use `${owner}/${repo}` instead?